kv/txn_coord_sender.go

```diff
@@ -322,8 +322,7 @@ func (tc *TxnCoordSender) Send(ctx context.Context, ba proto.BatchRequest) (*pro
 	// TxnID (if applicable). Begin a Trace which follows this request.
 	trace := tc.tracer.NewTrace(&ba)
 	defer trace.Finalize()
-	// TODO(tschottdorf): always "Batch"
-	defer trace.Epoch(fmt.Sprintf("sending %s", ba.Method()))()
+	defer trace.Epoch(fmt.Sprintf("sending %s", proto.Batch))()
```

Comment:
I've replaced uses of `proto.Batch` almost everywhere. There remain, I think, two places where a TODO is in place to get rid of them already.